### PR TITLE
Normalize gax support

### PR DESCRIPTION
**Pull request by tseaver**

### CODE REVIEW

File: gcloud/logging/_gax.py [hunk 1]
```diff
@@ -413,16 +412,9 @@ def metric_delete(self, project, metric_name):
...
-    if page_token is None:
-        page_token = INITIAL_PAGE
-    options = {'page_token': page_token}
-    return CallOptions(**options)
-
-
 def _mon_resource_pb_to_mapping(resource_pb):
-    """Helper for  :func:_log_entry_pb_to_mapping"""
+    """Helper for  :func:_log_entry_pb_to_mapping
+    """
```
Comment (dhermes) [inline gcloud/logging/_gax.py]: Please no dangling `"""` and also use a backtick after `:func:` (and to close the function reference.

File: gcloud/logging/_gax.py [hunk 2]
```diff
@@ -467,17 +457,20 @@ def _value_pb_to_value(value_pb):
...
     return dict([(key, _value_pb_to_value(struct_pb.fields[key]))
                  for key in struct_pb.fields])
 
 
 def _log_entry_pb_to_mapping(entry_pb):
     """Helper for :meth:`list_entries`, et aliae
 
-    Ideally, would use a function from :mod:`protobuf.json_format`, but
-    the right one isn't public.  See:
-    https://github.com/google/protobuf/issues/1351
```
Comment (dhermes) [inline gcloud/logging/_gax.py]: This message no longer worth keeping around?

**Comment by dhermes** [comment]
@tseaver What inspired this hygienic PR?

File: gcloud/logging/_gax.py (hunk 2 above)
Comment (tseaver) [inline gcloud/logging/_gax.py]: Yup, the `json_format` bit wouldn't help much, as we'd still have to map all the `with_undersoce` name to camelCase for equivalence with the JSON API.

File: gcloud/logging/_gax.py (hunk 1 above)
Comment (tseaver) [inline gcloud/logging/_gax.py]: I missed pasting in the "impedance matching" comment.

**Comment by dhermes** [comment]
LGTM.

**Pull request merged**